docs/workflows/batch_processing/about.md

```diff
@@ -87,13 +87,125 @@ inference rf-cloud data-staging create-batch-of-images --images-dir <your-images
 
 for videos:
 ```bash
-inference rf-cloud data-staging create-batch-of-videos --videos-dir <your-images-dir-path> --batch-id <your-batch-id>
+inference rf-cloud data-staging create-batch-of-videos --videos-dir <your-videos-dir-path> --batch-id <your-batch-id>
 ```
 
 !!! hint "Format of `<your-batch-id>`"
 
     Batch ID must be lower-cased string without special caraters, with letters and digits allowed.
 
+#### Cloud Storage Integration
+
+If your data is already stored in cloud storage (S3, Google Cloud Storage, or Azure), you can process it directly without downloading files locally. This feature generates presigned URLs for your cloud files, making it efficient for large datasets.
+
+!!! info "Installing Cloud Storage Support"
+
+    Cloud storage integration requires additional dependencies. Install them with:
+
+    ```bash
+    pip install 'inference-cli[cloud-storage]'
+    ```
+
+**For images stored in cloud storage:**
+
+```bash
+inference rf-cloud data-staging create-batch-of-images \
+  --data-source cloud-storage \
+  --bucket-path <cloud-path> \
+  --batch-id <your-batch-id>
+```
+
+**For videos stored in cloud storage:**
+
+```bash
+inference rf-cloud data-staging create-batch-of-videos \
+  --data-source cloud-storage \
+  --bucket-path <cloud-path> \
+  --batch-id <your-batch-id>
+```
+
+The `--bucket-path` parameter supports:
+- **S3**: `s3://bucket-name/path/`
+- **Google Cloud Storage**: `gs://bucket-name/path/`
+- **Azure Blob Storage**: `az://container-name/path/`
+
+You can optionally include glob patterns to filter files:
+- `s3://my-bucket/training-data/**/*.jpg` - All JPG files recursively
+- `gs://my-bucket/videos/2024-*/*.mp4` - MP4 files in 2024-* folders
+- `az://container/images/*.png` - PNG files in images folder
+
+!!! hint "Cloud Storage Examples"
+
```

Comment:
should highlight here that the credtials are used only locally byt the CLI tool to generate signed URLs